README.md

```diff
@@ -1,2 +1,33 @@
-# msi
-Use Magento Multi Source Inventory for stock.
+# Rapidez MSI
+This Rapidez Module will offer compatibility with the Magento MSI functionality.
+This module uses a HTTP Middleware to have access to the ```stock_id ``` for that website. 
+
+It also features 2 Global Scopes used by [Eventy Filters](https://docs.rapidez.io/0.x/package-development.html#eventy-filters) to enrich product Detail and Overview Page and The [Rapidez:Indexer](https://docs.rapidez.io/0.x/indexer.html#indexer) with Stock Status Information.
+
+##Requirements
+This modules requires the Magento MSI functionality to be active and fully functional on the Magento side.
+Stock(s) and Source(s) should be created and assigned to the Website(s).
+
+The [Message Queues](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/message-queues/message-queues.html) of Magento should be functional as MSI stock tables will be update through this mechanism. For example, the salable status of a product will be updated by process ` inventory.reservations.updateSalabilityStatus`.
+
+##Installation
+```composer require rapidez/msi```
+
+To expose the `stock_qty` to the product detail and overview page you can add the following configuration to the ```config/rapidez.php``` file:
```

Comment:
My preference here would go out to this module having it's own config file. e.g. msi.php
You can see an example here https://github.com/rapidez/strapi/blob/master/src/StrapiServiceProvider.php
where it loads the config from /config/strapi.php and if you publish the config you can override those values

Reply:
You would still need to do what @rbnmulder says here to have this var available in the JS. look here: https://github.com/rapidez/core/blob/master/config/rapidez.php#L23

Reply:
@indykoning I made an msi config.